libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/transition/MapboxNavigationCameraTransition.kt

```diff
@@ -248,7 +248,7 @@ class MapboxNavigationCameraTransition(
 
         cameraOptions.bearing?.let { bearing ->
             var bearingShortestRotation = bearing
-            currentMapCamera.bearing?.let {
+            currentMapCameraState.bearing.let {
```

Comment:
null check unnecessary

Reply:
Sorry, no null check here, but the whole `let` construct is a little bit redundant here now, same above.